### PR TITLE
cEURUSDC cBRLUSDC Parameters

### PR DESCRIPTION
**Pull request by sissnad**
### Description

_A few sentences describing the overall effects and goals of the pull request's commits.
What is the current behavior, and what is the updated/expected behavior with this PR?_

### Other changes

_Describe any minor or "drive-by" changes here._

### Tested

_An explanation of how the changes were tested or an explanation as to why they don't need to be._

### Related issues

- Fixes #[issue number here]

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### Documentation

_The set of community facing docs that have been added/modified because of this change_

### CODE REVIEW

File: script/upgrades/MU01/governance/Config.sol [hunk 1]
```diff
@@ -354,4 +359,68 @@ library Config {
...
+      isConstantSum: true,
+      spread: FixidityLib.newFixedFraction(25, 10000), // 0.0025
+      minimumReports: 5,
+      referenceRateResetFrequency: 5 minutes,
+      stablePoolResetSize: 1_800_000 * 1e18, // 12 million
+      isMedianDeltaBreakerEnabled: true,
+      medianDeltaBreakerThreshold: FixidityLib.newFixedFraction(2, 100), // 0.02
+      medianDeltaBreakerCooldown: 15 minutes,
+      smoothingFactor: FixidityLib.newFixedFraction(5, 10000), // 0.0005
+      isValueDeltaBreakerEnabled: true,
```
Comment (sissnad) [inline script/upgrades/MU01/governance/Config.sol:377]: @ninocomputer @bayological `cEURUSDC` `valueDeltaBreaker` should be referencing `USDCUSD`. I don't know the name of  the parameter.

File: script/upgrades/MU01/governance/Config.sol [hunk 2]
```diff
@@ -354,4 +359,68 @@ library Config {
...
+      isConstantSum: true,
+      spread: FixidityLib.newFixedFraction(25, 10000), // 0.0025
+      minimumReports: 5,
+      referenceRateResetFrequency: 5 minutes,
+      stablePoolResetSize: 1_800_000 * 1e18, // 12 million
+      isMedianDeltaBreakerEnabled: true,
+      medianDeltaBreakerThreshold: FixidityLib.newFixedFraction(25, 1000), 0.025
+      medianDeltaBreakerCooldown: 15 minutes,
+      smoothingFactor: FixidityLib.newFixedFraction(5, 10000), //0.0005 
+      isValueDeltaBreakerEnabled: true,
```
Comment (sissnad) [inline script/upgrades/MU01/governance/Config.sol:409]: @ninocomputer @bayological `cBRLUSDC` `valueDeltaBreaker` should be referencing `USDCUSD`. I don't know the name of  the parameter.

File: script/upgrades/MU01/governance/Config.sol (hunk 1 above)
Comment (ninocomputer) [reply script/upgrades/MU01/governance/Config.sol:377]: rateFeedDependencies

File: script/upgrades/MU01/governance/Config.sol (hunk 2 above)
Comment (ninocomputer) [reply script/upgrades/MU01/governance/Config.sol:409]: did we decide that we should reference this in pool configs

Comment (sissnad) [reply script/upgrades/MU01/governance/Config.sol:409]: I am not sure. Just want to make sure it's not missed

**Pull request closed without merging**